package.json

```diff
@@ -15,25 +15,26 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "pretest": "npm run lint && npm run prettier",
+    "pretest": "npm run lint",
     "prettier": "prettier --list-different --write \"./**/**.{js,jsx}\"",
     "release": "conventional-changelog -i CHANGELOG.md -s",
     "test": "jest --coverage"
   },
   "license": "MIT",
   "dependencies": {
-    "@apidevtools/json-schema-ref-parser": "^9.0.1",
-    "deep-extend": "^0.6.0",
-    "r2": "^2.0.1",
+    "@apidevtools/json-schema-ref-parser": "^9.0.7",
+    "node-fetch": "^2.6.1",
     "swagger-parser": "^10.0.1",
-    "swagger2openapi": "^2.11.16",
+    "swagger2openapi": "^7.0.6",
```

Comment:
omfg lol

Reply:
I've been getting pinged by Dependabot about this every month when it tries to update it. 😅